syntax = "proto3";

enum OperationStatus {
  SUCCESS = 0;
  NOT_ENOUGH_COINS = 1;
  SELF_TRANSACTION = 2;
  USER_NOT_FOUND = 3;
  USER_ALREADY_EXISTS = 4;
  STORAGE_IS_FULL = 5;
}